app/javascript/availability/libraries_locations.json

```diff
@@ -671,7 +678,7 @@
     "STATUTE-DN": "Dickinson Law (Carlisle) - 2nd Floor - Statutes",
     "STATUTE-DP": "Penn State Law (UP) - 2nd Floor - Statutes",
     "STORAGE-FE": "Fayette - In Storage, Ask at Circ. Desk - In-Library Use",
-    "STORAGE-HN": "Copy unavailable, request via Interlibrary Loan",
+    "STORAGE-HN": "Hazleton",
```

Comment:
same for this location, it is in the ill list and the label change here won't have an effect.

Reply:
now moved out of ILL list